modules/docs/mdx/8.0-UPGRADE-GUIDE.mdx

```diff
@@ -178,7 +178,76 @@ const Example = styled('div')(
 
 ### Responsive Styles
 
-> Coming Soon (https://github.com/Workday/canvas-kit/pull/1779)
+The following Canvas Kit responsive utilities will be released as part of v8:
+
+- **ResponsiveContextProvider**: This is a React Context Provider that will take a width and an
+  optional theme and allow components to subscribe to context changes via the `useResponsiveContext`
+  hook.
+
+- **useResponsiveContext Hook**: This hook subscribes to the `ResponsiveContextProvider` and
+  provides helpful utilities to determine whether the current width is within a particular theme
+  breakpoint. Developers can use these utilities when creating container-based responsive styles.
```

Comment:
should we reference what those  default theme breakpoints are?

Reply:
Yes we should. I'll update